code/modules/chemistry/Reagents-Medical.dm

```diff
@@ -666,14 +666,26 @@ datum
 					var/mob/M = holder.my_atom
 					REMOVE_ATOM_PROPERTY(M, PROP_MOB_STAMINA_REGEN_BONUS, "r_smelling_salt")
 				..()
+			reaction_mob(var/mob/M, var/method=TOUCH, var/volume_passed, var/list/paramslist = 0)
+				if(method == INGEST && volume_passed >= 3)
+					if(isliving(M) && !M.hasStatus("smelling_salts") && ("inhaled" in paramslist))
+						var/mob/living/H = M
+						H.delStatus("drowsy")
+						H.delStatus("passing_out")
+						if (H.stamina < 0 || H.hasStatus("weakened")) //enhanced effects if you're K/Oed (also implies a second person is applying this)
+							H.TakeDamage("chest", 0, 10, 0, DAMAGE_BURN) // a little damage penalty
+							if (H.use_stamina)
+								H.stamina = max(H.stamina_max*0.2,H.stamina)
+							H.changeStatus("weakened", -20 SECONDS)
```

Comment:
are you sure you want to be checking for `"weakened"`? `"paralysis"` is the "you are knocked out and asleep" status

Reply:
weakened is predominantly what gets you downed in crit and from punches to the face - paralysis is odd as it covers stuff I wouldn't ordinarily want this to work on, like literal paralysants.

after re-reading this i think that given it's a one-off effect it *could* work on both, as full-on paralysants also tend to leave you with >20s of down time. your th oughts?

Reply:
could go either way - just that the comment shouldn't say "K/Oed" if it's not for knockouts lol